lib/src/migrators/module.dart

```diff
@@ -42,9 +54,15 @@ class ModuleMigrator extends Migrator {
   /// If [migrateDependencies] is false, the migrator will still be run on
   /// dependencies, but they will be excluded from the resulting map.
   Map<Uri, String> migrateFile(Uri entrypoint) {
+    if (argResults['forward'] == 'prefixed' &&
+        argResults['remove-prefix'] == null) {
+      throw MigrationException(
+          'Error: Must provide --remove-prefix when --forward=prefixed');
```

Comment:
I might make this a little clearer about *why* this is necessary. Maybe "You must provide --remove-prefix with --forward=prefixed so we know which prefixed members to forward" or something like that?

Also, I'd avoid putting "Error: " at the beginning of literal exception messages. If you want a header like that, it should be the responsibility of the error handler that catches and prints the exceptions.